src/__tests__/addDisplayName.coffee

```diff
@@ -42,20 +42,20 @@ WithBranch = flowMax(
 
 WithAddWrapper = flowMax(
   addDisplayName 'WithAddWrapper'
-  addWrapper ({render: _render}) -> _render()
+  addWrapper (_render) -> _render()
```

Comment:
Unrelated - there was a silent merge conflict when merging #56 (since it was based on a branch that didn't include the breaking change to `addWrapper()` yet)